### DISCUSSION TITLE
documentation for starting out

### DISCUSSION THREAD

**Discussion by rmschindler** [11ty/eleventy#3263] [Uncategorized]
I’m just starting out with Eleventy, and I’m looking at the docs (`www.11ty.dev`), and I am finding the conspicuous absence of any kind of description of how 11ty works fundamentally . what I mean is this: something describing what goes where in order to do what . hmm... I’m struggling to find the words to express the predicament I am feeling: it’s like the documentation is skirting around simply directly saying what is going on . I read somewhere that it’s ‘convention over configuration’ — I like that — but then: what are the conventions? . or do I need to be like a detective, and read all of the docs, and then deduce for myself what the conventions are?

perhaps I am missing a resource that is well-known-about . I hope that is the case . if you know of such a resource, I would appreciate your letting me know

well, that’s all I guess . I’m excited about 11ty, but baffled at the docs which seem only indirectly to tell me how to configure and use it

**Comment by bobmonsour** [comment]
Here’s a resource that might help. 

https://11tybundle.dev/categories/getting-started/
